rust/windows-client/src-tauri/src/client.rs

```diff
@@ -32,6 +32,7 @@ pub(crate) struct AppLocalDataDir(std::path::PathBuf);
 const CREATE_NO_WINDOW: u32 = 0x08000000;
 
 pub(crate) fn run() -> Result<()> {
+    std::panic::set_hook(Box::new(tracing_panic::panic_hook));
```

Comment:
It's still good to bubble up the panic through ffi in case that the tracing logger isn't available at the point of the crash.

Reply:
I'm not sure what to do, then. I was thinking I should set a panic hook when the GUI starts, even if we don't sign in and start connlib.

I should somehow log panics while the GUI is signed out, right?

Reply:
Hm, it's okay to leave this `set_hook` for that case but you'll need to override it each time after sign out.

I think that's such a rare edge-case that we shouldn't worry about, most panics would be while connlib *is* running and depend on the window's crash report for these cases cc @jamilbk. 

The most important part of this is modifying the `set_hook` in connlib to use `tracing::error`

Reply:
My bad, you don't need to override it after each sign out since connlib's panic hook would still work. So we can leave it as this and we'd still get the panic in the logs before sign in and thereafter connlib's would be used.

Reply:
@conectado what did you mean by "bubble up the panic through ffi" by the way?

Reply:
oh I just mean that there's still a point on calling `on_disconnect` with the panic error as we are doing right now even if we log it.